src/GeositeFramework/css/main.css

```diff
@@ -2335,3 +2335,7 @@ body.x-body {
 .launchpad-plugin li p {
   font-size: 13px;
 }
+.btn-default.dropdown-toggle#map-utils-dropdown-button > .icon-wrench:before {
```

Comment:
Consider using the selector:

```css
#map-utils-dropdown-button > i:before
```

Since ids are unique, they need no further qualification. By targeting `i` rather than `.icon-wrench`, we ensure that the placement will stick even if the specific icon is changed down the line.